range_min : 0.1
range_max : 100.0

angle_min : -180.0
angle_max : 180.0
z_min : -0.7
z_max : 10.0
y_min : 0.1
x_min : 0.1

